docs/_docs/Starters Academy (LDSSA)/01-Starters-Academy-(Course).md

```diff
@@ -103,7 +101,7 @@ top-performing solution, but a good set of baseline steps and techniques that ma
 This instructor solution will be published on the day
 after the hackathon so that students can study it and iterate on it on at their own pace.
 
-There are 5 hackathons in total, held on selected Sundays throughout the academy. To attend
+There are 6 hackathons in total, held on selected Sundays throughout the academy. To attend
```

Comment:
So, there are 5 specializations, but 6 hackathons in total?

Reply:
6 hackathons, thank you for catching this.